plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/BuildPath.java

```diff
@@ -158,7 +158,11 @@ public static Library collectLibraryFiles(IJavaProject javaProject, List<Library
     
     List<LibraryFile> resolved = Library.resolveDuplicates(new ArrayList<LibraryFile>(masterFiles));
     subMonitor.worked(8);
+    
     masterLibrary.setLibraryFiles(resolved);
+    
+    masterLibrary.setResolved();
```

Comment:
So, transitive dependency versions get updated without this? (I fail to follow the overall pipeline.)

Reply:
Yes, exactly, which we don't want to happen. We want the latest version of objectify, but if the latest version of objectify asks for Guava 20.0, we don't want to give it Guava 23 instead which is what was happening prior to this PR.